microsoft-edge/webview2/release-notes.md

```diff
@@ -3,7 +3,7 @@ description: Release notes for Microsoft Edge WebView2 SDK
 title: Release Notes for Microsoft Edge WebView2 for Win32, WPF, and WinForms
```

Comment:
The YAML "title:" string (it populates the browser tab) should match the h1 text (top-of-page title).  It looks like the title: and description: strings need to be swapped.
@vchapel

Reply:
fixing